scripts/push-daemon.sh

```diff
@@ -0,0 +1,10 @@
+set -e
+set -o pipefail
+
+DOCKER_IMAGE_TAG=$(cat /tmp/docker_image_tag)
```

Comment:
Maybe we should check whether this is not empty, just in case, and return error if it is.

Someone could try to run locally without running the build first, for instance.
Or someone could mess with the build and break the right order of execution of scripts without noticing.